pkg/man

```diff
@@ -1,3 +1,4 @@
 [build]
+cp "$K"/manpp "$butch_root_dir""$butch_prefix"/bin
```

Comment:
this will break since the destination dir does not exist yet
please use install -Dm 755 instead of cp and do it after the compile, so it only gets installed if the compilation succeeds.